packages/lit-dev-tools-cjs/src/api-docs/transformer.ts

```diff
@@ -568,6 +573,85 @@ export class ApiDocsTransformer {
     return a.name.localeCompare(b.name);
   };
 
+  /**
+   * Temporarily turn more modern formats of TypeDoc into a legacy format that
+   * our HTML template understands.
+   */
+  private replaceCommentNodesWithLegacyCommentFields(
```

Comment:
This name might also be better as something along the lines of. replaceCommentNodePartsWithMarkdown?